modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/OLSLoad_conf.pm

```diff
@@ -76,7 +76,7 @@ sub default_options {
         copy_service_mart_payload =>
```

Comment:
Can we change the name of payload and name of analysis to see this is the ontology_mart copy, say `copy_ontology_mart`
Same change applies to the other copy which could be renamed to `copy_ensembl_ontology`